src/styles/getNavigationModalCardStyles/index.website.js

```diff
@@ -7,5 +7,8 @@ export default ({windowHeight, isSmallScreenWidth}) => ({
     // Safari issues:
     // https://github.com/Expensify/App/issues/12005
     // https://github.com/Expensify/App/issues/17824
+    // https://github.com/Expensify/App/issues/20709
+
     height: `${windowHeight}px`,
+    minHeight: `${windowHeight}px`,
```

Comment:
This caused a [regression](https://github.com/Expensify/App/issues/23481) on Safari where the window height is not reliable.